tests/get_kv_table_nodeos_tests.cpp

```diff
@@ -230,6 +230,45 @@ BOOST_FIXTURE_TEST_CASE( get_kv_table_nodeos_test, TESTER ) try {
    chk_result(3, 9);
    chk_result(4, 10);
 
+   // test next_key
```

Comment:
all BOOST_REQUIRE_EQUAL(result.next_key_bytes != "", true); checks should be changed to verifying the expected behavior
I know this isn't your fault, but the test should be verifying the value, and we should be verifying the result.next_key value as well. If any of the cases you added are repeats of existing cases then we should remove them and just add the result.next_key check in the existing one. But make sure that we don't lose the checks I added above for boundary conditions like "dec" being "10" and "hex" being "a"(or "A") when the uinXX_t has a value of 0x0a.

For any of your new cases that stays around, also add verifying the result.next_key_bytes value as well.

Reply:
> For any of your new cases that stays around, also add verifying the result.next_key_bytes value as well.

Should we verify the actual value of result.next_key_bytes or verify its emptiness? It seems to me that checking actual value doesn't help much because a value of bytes type is an encoded hex.

Reply:
> For any of your new cases that stays around, also add verifying the result.next_key_bytes value as well.

Fixed.